docs/howto/connection/index.rst

```diff
@@ -81,6 +81,38 @@ in the scheme part of URI, so it must be changed to a hyphen character
 (e.g. ``google-compute-platform`` if ``conn_type`` is ``google_compute_platform``).
 Query parameters are parsed to one-dimensional dict and then used to fill extra.
 
+Generating a connection URI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Building the URI for a connection can be tricky.  To make it easier, the
+:py:class:`~airflow.models.connection.Connection` class has a convenience method
+:py:meth:`~airflow.models.connection.Connection.get_uri`.  It can be used like so:
+
+.. code-block:: python
```

Comment:
```suggestion
.. code-block:: pycon
```
Reference: https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

Reply:
cool! thank you will do this